docs/content/preview/explore/query-1-performance/pg-stat-progress-copy.md

```diff
@@ -0,0 +1,177 @@
+
+| Column | Description |
+| :---- | :---------- |
+| pid | Process ID of backend. |
+| datid | OID of the database to which this backend is connected. |
+| datname | Name of the database to which this backend is connected. |
+| relid | OID of the table on which the COPY command is executed. It is set to 0 if copying from a SELECT query.|
+| command | The command that is running COPY FROM, or COPY TO. |
+| type | The I/O type that the data is read from or written to: FILE, PROGRAM, PIPE (for COPY FROM STDIN and COPY TO STDOUT), or CALLBACK (used for example during the initial table synchronization in logical replication). |
+| yb_status | Tracking status of the copy command. |
+| bytes_processed | Number of bytes already processed by COPY command. |
+| bytes_total | Size of the source file for COPY FROM command in bytes. It is set to 0 if not available. |
+| tuples_processed | Number of tuples already processed by COPY command. |
+| tuples_excluded | Number of tuples not processed because they were excluded by the WHERE clause of the COPY command. |
+
+## YugabyteDB specific changes
+
+The `pg_stat_progress_copy` view includes the following changes which are YugabyteDB specific:
+
+### Definition of `tuples_processed`
+
+The definition of `tuples_processed` column is different in YugabyteDB in comparison to PostgreSQL.
+
+For the `COPY` command in YugabyteDB, an option `ROWS_PER_TRANSACTION` is added which defines the transaction size to be used by the `COPY` command. For example, if the total tuples to be copied are 5000 and `ROWS_PER_TRANSACTION` is set to 1000, then the database will create 5 transactions and each transaction will insert 1000 rows. If there is an error during the execution of the copy command, then some tuples can be persisted based on the already completed transaction. This also implies that if the error occurs after inserting the 3500th row, then the first 3000 rows will still be persisted in the database.
+
+- 1 to 1000 →  Transaction_1
+- 1001 to 2000 → Transaction_2
+- 2001 to 3000 → Transaction_3
+- 3001 to 3500 → Error
+
+First 3000 rows will be persisted to the table and `tuples_processed` will show 3000.
+
+Because the `copy` command is divided into multiple transactions, `tuples_processed` tracks the rows for which the transaction has already completed.
+
+### New column `yb_status`
+
+The `pg_stat_progress_copy` view includes a status column `yb_status` to indicate the status of the copy command.
+
+If a `COPY` command is terminated due to any error, then it is possible that some tuples are persisted as explained in the [`tuples_processed`](#definition-of-tuples-processed) section. In such scenarios, `tuples_processed` shows a non-zero count and `yb_status` will show that the `COPY` command was terminated due to an error. In PostgreSQL, it is not required as copy is one single transaction. However, for YugabyteDB, without this column, you can't determine if the copy successfully completed or not.
```

Comment:
`However, for YugabyteDB, without this column, you can't determine if the copy successfully completed or not.` 

Rewrite as follows -- 

For YugabyteDB  yb_status columns helps finding   if the copy successfully completed or not.